hazelcast-enterprise/Dockerfile

```diff
@@ -46,10 +48,11 @@ RUN echo "Installing new APK packages" \
     && echo "Downloading Hazelcast and related JARs" \
     && mkdir "${HZ_HOME}/lib" \
     && cd "${HZ_HOME}/lib" \
-    && for JAR_URL in ${HAZELCAST_ALL_URL} ${CACHE_API_URL} ${PROMETHEUS_AGENT_URL} ${NETTY_URL} ${NETTY_TCNATIVE_URL} ${EUREKA_PLUGIN_URLS}; do curl -sf -O -L ${JAR_URL} || exit $?; done \
+    && for JAR_URL in ${HAZELCAST_ALL_URL} ${CACHE_API_URL} ${PROMETHEUS_AGENT_URL} ${NETTY_URL} ${NETTY_TCNATIVE_URL} ${EUREKA_PLUGIN_URLS} ${LOG4J2_URLS}; do curl -sf -O -L ${JAR_URL} || exit $?; done \
     && mv jmx_prometheus_javaagent-*.jar jmx_prometheus_javaagent.jar \
-    && echo "Granting read permission to ${HZ_HOME}" \
+    && echo "Granting read and write permissions to ${HZ_HOME}" \
     && chmod -R +r ${HZ_HOME} \
+    && chmod o+w ${HZ_HOME} \
```

Comment:
Why `write` permission is required?

Reply:
Otherwise `sed` fails during in-place editing at:
```
if [ -n "${LOGGING_LEVEL}" ]; then
  sed -i "s/rootLogger.level=info/rootLogger.level=${LOGGING_LEVEL}/g" log4j2.properties
fi
```
The currently documented command also faces this problem when it's not run as root.

Reply:
Removed with environment variable sol'n.